From ee56e834e65f8017f7433cb9911ddf96f498ea15 Mon Sep 17 00:00:00 2001
From: Oleh Ivaniuk <fedencer7@gmail.com>
Date: Thu, 5 Sep 2024 18:10:10 +0300
Subject: [PATCH] Expanded list of method argument names

---
 src/transactions.rs | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/src/transactions.rs b/src/transactions.rs
index 4a5ec3f..6673403 100644
--- a/src/transactions.rs
+++ b/src/transactions.rs
@@ -33,7 +33,7 @@ const EVENT_JSON_PREFIX: &str = "EVENT_JSON:";
 const BLOCK_HEADER_CLEANUP: u64 = 2000;
 const MAX_COMMIT_HANDLERS: usize = 3;
 
-const POTENTIAL_ACCOUNT_ARGS: [&str; 19] = [
+const POTENTIAL_ACCOUNT_ARGS: [&str; 21] = [
     "receiver_id",
     "account_id",
     "sender_id",
@@ -53,6 +53,8 @@ const POTENTIAL_ACCOUNT_ARGS: [&str; 19] = [
     "beneficiary_id",
     "staking_pool_account_id",
     "owner_account_id",
+    "claimer",
+    "bounty_owner",
 ];
 
 const POTENTIAL_EVENTS_ARGS: [&str; 10] = [
-- 
2.34.1

